source/_integrations/modbus.markdown

```diff
@@ -626,6 +626,11 @@ climates:
           description: "Address of Fan mode register."
           required: true
           type: integer
+        write_registers:
```

Comment:
As requested in the code, please do not add a new parameter....make the address an int or an array.